***Started - reading and parsing pddl file ...***
***Finished reading and parsing pddl file after 0.147 seconds CPU time, 0.141 seconds wall-clock time***

***Started - preprocessing tarski problem ...***
***Finished preprocessing tarski problem after 0.000 seconds CPU time, 0.000 seconds wall-clock time***

***Started - grounding ...***
/tmp/tmpn3kasmp2:80:17-18: error: syntax error, unexpected (

/tmp/tmpn3kasmp2:81:28-29: error: syntax error, unexpected (

*** ERROR: (gringo): grounding stopped because of errors
Traceback (most recent call last):
  File "brfs.py", line 60, in <module>
    main( sys.argv[1], sys.argv[2], sys.argv[3] )
  File "brfs.py", line 15, in main
    ground_generate_task( domain_file, problem_file, task )
  File "/home/anubhav-cs/Documents/Research_Project/anubhav-research/LAPKT-public/planners/brfs-tarski/tarski_parser/tarski_util.py", line 185, in ground_generate_task
    reachable_action_params =   copy.deepcopy( grounding.ground_actions())
  File "/media/anubhav-cs/SSD256/Applications/tarski/src/tarski/grounding/lp_grounding.py", line 39, in ground_actions
    model = self._solve_lp()
  File "/media/anubhav-cs/SSD256/Applications/tarski/src/tarski/grounding/lp_grounding.py", line 50, in _solve_lp
    model_filename = run_clingo(lp)
  File "/media/anubhav-cs/SSD256/Applications/tarski/src/tarski/reachability/clingo_wrapper.py", line 26, in run_clingo
    raise ExternalCommandError("Gringo exited with code {}".format(retcode))
tarski.errors.ExternalCommandError: Gringo exited with code 1
